examples/cull-idle/cull_idle_servers.py

```diff
@@ -127,6 +127,19 @@ def handle_server(user, server_name, server):
                 log_name, server['pending'])
             return False
 
+        # jupyterhub < 0.9 defined 'server.url' once the server was ready
+        # as an *implicit* signal that the server was ready.
+        # 0.9 adds a dedicated, explicit 'ready' field.
+        # By current (0.9) definitions, servers that have no pending
+        # events and are not ready shouldn't be in the model,
+        # but let's check just to be safe.
+
+        if not server.get('ready', bool(server['url'])):
```

Comment:
We may wish to be more explicit with the warning, but we can do that if folks experience in beta.